test/evergreen.yml

```diff
@@ -1990,6 +1990,48 @@ tasks:
           # No of processes to run in the background
           no_of_procs: 10
 
+  # The task name is ppc-zseries because this task will be used in both buildVariants
```

Comment:
I just had a look at the `wiredtiger-test-format-stress-ppc` and `wiredtiger-test-format-stress-zseries` in Jenkins and I can't see there is a similarity between two tests. Can you double check if one task will work in both build variants or do we need seperate tasks?